alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultNotificationAccessor.java

```diff
@@ -185,6 +193,25 @@ public PageRequest getPageRequestForNotifications(Integer pageNumber, Integer pa
         return PageRequest.of(pageNumber, pageSize, Sort.by(sortingOrder));
     }
 
+    //TODO this needs unit tests
+    @Override
+    public Page<AlertNotificationModel> findNotificationsNotProcessed() {
+        Sort.Order sortingOrder = Sort.Order.asc("providerCreationTime");
+        PageRequest pageRequest = PageRequest.of(0, 100, Sort.by(sortingOrder));
+        return notificationContentRepository.findNotProcessedNotifications(pageRequest)
+                   .map(this::toModel);
+    }
```

Comment:
The sort and page size should be reflected in the method name. Also, I believe we don't want to expose `Page` (use `AlertPagedModel` instead?) outside of the DB API, but we may not have done the best job of that up until this point.